chevah/compat/tests/normal/testing/test_assertion.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2015 Adi Roiban.
+# See LICENSE for details.
+"""
+Tests for the assertion helpers.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
```

Comment:
well... this is the code from imported from empirical, where instead of unicode_literals we went for `builtin str`